src/helpers/map.js

```diff
@@ -198,9 +138,38 @@ export function getTraversableCells(cpt, map, me) {
   for (x = 0; x < xl; x++) {
     for (let y = 0; y < yl; y++) {
       const objRadius = 20; // TODO: Set radius to be correct value for each cell object
-      fillGridWithSubgrid(emptyCells, traversableCellsInTile(isTraversable(map[x][y], me),
+      fillGridWithSubgrid(emptyCells, traversableCellsInTile(isTraversable(map[x][y]),
         cpt, objRadius), x * cpt, y * cpt);
     }
   }
   return emptyCells;
 }
+
+/*
+ * Returns the position (in pixels x,y and grid positions xg, yg
+ * of first of the specified tile type to appear starting in the
+ * top left corner and moving in a page-reading fashion.
+ */
+export function findTile(targetTile) {
+  for (let x = 0, xl = tagpro.map.length, yl = tagpro.map[0].length; x < xl; x++) {
+    for (let y = 0; y < yl; y++) {
+      if (tagpro.map[x][y] === targetTile) {
+        return { x: x * PIXELS_PER_TILE, y: y * PIXELS_PER_TILE, xg: x, yg: y };
+      }
+    }
+  }
+  console.error(`Unable to find tile: ${targetTile}`);
+  return {};
+}
+
+export function findApproxTile(targetTile) {
```

Comment:
Yeah I can document in a comment, it's used for when we don't care whether a flag is taken or not, or if a bomb is active or not.